include/dali/core/util.h

```diff
@@ -84,6 +84,23 @@ constexpr Value align_up(Value v, Alignment a) {
   return v + ((a - 1) & -v);
 }
 
+template <typename Value, typename Alignment>
+DALI_HOST_DEV
+constexpr Value align_reminder(Value v, Alignment a) {
+  return v & (a - 1);
+}
+
+template <typename Value, typename Alignment>
+DALI_HOST_DEV
+constexpr Value align_down(Value v, Alignment a) {
+  return v & ~(a - 1);
```

Comment:
One instruction fewer:
```suggestion
  return v & -a;
```

Reply:
Done